### PR TITLE
Update ChangeLog

### PR DESCRIPTION
**Pull request by vmcj**
Not sure if all of these things are relevant, but this is based on the commit messages of:
`git log --oneline 7.3.2..HEAD`

### CODE REVIEW

**Review by thijskh — APPROVED** [review]

File: ChangeLog [hunk 1]
```diff
@@ -42,7 +42,14 @@ Version 8.0.0 - DEV
    to convert it to the new style (see 'boolfind' example) or change it to
    include the old run_wrapper.
  - Replace jscolor by coloris.
-
+ - Created insights page in current work queue
+ - Increase performance by disabling class hotinjection
+ - Added alternative inplace install mode
+ - Added PHP8.0 support
+ - New judgedaemons can now be default paused
```
Comment (nickygerritsen) [inline ChangeLog:49]: This sentence doesn’t flow nicely

File: ChangeLog [hunk 2]
```diff
@@ -42,7 +42,14 @@ Version 8.0.0 - DEV
    to convert it to the new style (see 'boolfind' example) or change it to
    include the old run_wrapper.
  - Replace jscolor by coloris.
-
+ - Created insights page in current work queue
+ - Increase performance by disabling class hotinjection
```
Comment (nickygerritsen) [inline ChangeLog:46]: Maybe better: increase performance by not dynamically loading classes.

**Review by nickygerritsen — APPROVED** [review]
All seem good to add

File: ChangeLog (hunk 2 above)
Comment (vmcj) [reply ChangeLog:46]: I agree, will change it.

**Pull request merged**